build/platforms.bzl

```diff
@@ -36,6 +36,23 @@ UBI_PLATFORMS = {
     ],
 }
 
+CLIENT_PLATFORMS = {
```

Comment:
You need to add the CLIENT_PLATFORMS to all_platforms too: https://github.com/jetstack/cert-manager/blob/d768862c3f8cf6d3dc560536051eef35ab659c91/build/platforms.bzl#L65

Reply:
Added